aspnet-core/src/AbpCompanyName.AbpProjectName.Application/Users/Dto/UserMapProfile.cs

```diff
@@ -8,7 +8,10 @@ public class UserMapProfile : Profile
         public UserMapProfile()
         {
             CreateMap<UserDto, User>();
-            CreateMap<UserDto, User>().ForMember(x => x.Roles, opt => opt.Ignore());
+            CreateMap<UserDto, User>(MemberList.Source)
```

Comment:
Why need to add `MemberList.Source`? As I know, it's related to validation rather than indicating the property side on ignore.